docs/timelines.md

```diff
@@ -0,0 +1,141 @@
+---
+id: timelines
+title: Timelines introduction
+sidebar_label: Timelines introduction
+---
+
+Timelines allow you to create a precisely timed light show which you can either
+run from the console's internal timer or synchronise to an external timecode
+source. This can be useful for an opening sequence for a corporate
+event, a complex song in a show where the song is pre-recorded or run to 
+a click track or timecode, or a multimedia event run to a playback track. You
+can also use it to create [complicated one-shot effects](./timelines/timeline-options.md#fader-tab) for busking.
+
+> You can also create timed shows by creating timecoded cues in 
+[cue-lists](./cue-lists/cue-list-timing.md#running-a-cue-list-to-timecode), but Timeline 
+recording allows much easier creation and editing of a timed show.
+
+Timelines can be built either from cues or chases you have already recorded, or by creating
+new cues as you go along. You can record them in real time by operating the console
+or by adding cues manually at specific times. You can think of a timeline as an automatic replay
+system to push the faders and buttons on the console. The actions recorded in a timeline
+are called **triggers**.
+
+To show the Timeline View window for a playback containing a timeline, touch the playback 
+information area relating to the fader or press \<Open/View\> then the **select button** 
+of the playback.
+
+![Playback View of Timeline](/docs/images/Timeline-Window.png)
+
+Triggers
+------------------
+
+To program a timeline you record actions at specific times
+which can then be replayed by the console with the same timing. These
+are called **Triggers**.
+
+The following types of trigger can be recorded. You can record these actions in real time using **Live Record**
+or enter them as manual triggers at specified times using the touch screen.
+
+Trigger             | Action
+--------------------|----------
+Set Level           | Set playback to a level over a fade time. This is like pushing up the fader over 2 seconds for example.
```

Comment:
```suggestion
Set Level           | Sets the fader level of a playback. This can be done over time. This applies whether or not the playback is physically assigned to a fader.
```

Something like that. I think it is important to emphasise that the playback fade time is a different thing.

Reply:
or maybe "Sets the level of a playback over a time. This is similar to pushing up the fader over 2 seconds."

I think the example is helpful for understanding. It's also not technically setting the fader level, it's setting the playback level which is a different thing.

Reply:
What about "Fades up a playback to a set level over a time. This has the same effect as you pushing up a fader. If the playback has fade times programmed these also run."